src/Simplify/Evaluate.elm

```diff
@@ -153,3 +153,31 @@ getInt resources baseNode =
 
         _ ->
             Nothing
+
+
+getNumber : Infer.Resources a -> Node Expression -> Maybe Float
+getNumber resources baseNode =
+    let
+        unparenthesized : Node Expression
+        unparenthesized =
+            AstHelpers.removeParens baseNode
+    in
+    case getInt resources unparenthesized of
```

Comment:
We should ideally support floats as well, so that `List.sum [ a, 0.0, b ]` also gets simplified.